lib/client/ClientRenderingWorkflow.js

```diff
@@ -172,6 +168,22 @@ var ClientRenderingWorkflow = {
 
 };
 
+function promisify(method) {
```

Comment:
it seems like the wrong place for this. Why don't you just use `new Promise(reject, resolve)`